src/AnalyzerConfiguration/NI.ruleset

```diff
@@ -211,17 +196,19 @@
     <Rule Id="VSTHRD111" Action="None" />
     <Rule Id="VSTHRD200" Action="Warning" />
   </Rules>
-  <Rules AnalyzerId="NationalInstruments.LabVIEW.Tools.Analyzers" RuleNamespace="NationalInstruments.LabVIEW.Tools.Analyzers">
-    <Rule Id="CA1804" Action="Warning" />
-    <Rule Id="CA2218" Action="Warning" />
+  <Rules AnalyzerId="NI.CSharp.Analyzers" RuleNamespace="NationalInstruments.Analyzers">        
     <Rule Id="LRN001" Action="Warning" />
     <Rule Id="LRT001" Action="Warning" />
     <Rule Id="NI1001" Action="Warning" />
     <Rule Id="NI1004" Action="None" />
     <Rule Id="NI1005" Action="Warning" />
     <Rule Id="NI1006" Action="Warning" />
     <Rule Id="NI1007" Action="Warning" />
+    <Rule Id="NI1009" Action="Error" />
```

Comment:
Any reason why this is Error vs Warning?

Reply:
I'm just reflecting what the default values are in the analyzers. It really should be warning, but that should also be true in our analyzers as that's normally what we follow.

Reply:
There's no reason I have to reflect it though, so I'll change it to warning to be consistent